styles/style.css

```
@@ -0,0 +1,123 @@
+
+* {
+    font-family: 'Courier New', Courier, monospace
+}
```

You don't actually need to apply this to `*` applying it to `body` works just as well and is less likely to cause conflicts if you import multiple stylesheets:

```suggestion
body {
    font-family: 'Courier New', Courier, monospace
}
```